.github/workflows/test.yaml

```diff
@@ -7,7 +7,7 @@ on:
 
 jobs:
   snap-builds:
-    runs-on: ubuntu-16.04
+    runs-on: ubuntu-20.04
```

Comment:
this is fine, since this uses the snapcore/action-build GitHub action which uses LXD to build the snap anyways so we are still building the snap really on xenial as declared in the snapcraft.yaml for the snapd snap